src/main/scala/cromwell/engine/backend/local/SharedFileSystem.scala

```diff
@@ -56,14 +60,15 @@ object SharedFileSystem {
    * indirectly through one of its input expressions
    */
   private def localizePathViaSymbolicLink(call: Option[Call], originalPath: Path, executionPath: Path): Try[Unit] = {
-    call.flatMap(_.docker) match {
-      case Some(_) => Failure(new UnsupportedOperationException("Cannot localize with symbolic links with Docker"))
-      case _ => Try(Files.createSymbolicLink(executionPath, originalPath.toAbsolutePath))
-    }
+    if (call.flatMap(_.docker).isDefined)
+      Failure(new UnsupportedOperationException("Cannot localize with symbolic links with Docker"))
+    else if (originalPath.toFile.isDirectory)
+      Failure(new UnsupportedOperationException("Cannot localize directory with symbolic links"))
+    else Try(Files.createSymbolicLink(executionPath, originalPath.toAbsolutePath))
   }
 }
 
-trait SharedFileSystem {
+trait SharedFileSystem extends Backend {
```

Comment:
"The way I see it SharedFileSystem is a partial implementation of Backend"

Right, that's exactly what prompted my question. As I said, I think you're right that the thing to do here is to identify what that partial bit is and separate that out.